tests/test_procedure.py

```diff
@@ -204,7 +203,7 @@ def test_treatment_not_started(self):
         for code, display, system in not_started_codes:
             self.add_procedure(code, display, system)
             self.test_user = db.session.merge(self.test_user)
-            assert (known_treatment_not_started(self.test_user),
-                    "treatment '{}' didn't show as not started".format(
-                        (system, code)))
+            assert known_treatment_not_started(self.test_user),\
+                "treatment '{}' didn't show as not started".format(
```

Comment:
What's the function of this line? I don't think it should be part of the `assert`...